cpp/src/ast/transform.cu

```diff
@@ -138,24 +136,23 @@ std::unique_ptr<column> compute_column(table_view const table,
     cudf::mutable_column_device_view::create(output_column->mutable_view(), stream);
 
   // Configure kernel parameters
-  auto const num_intermediates     = expr_linearizer.get_intermediate_count();
-  auto const shmem_size_per_thread = static_cast<int>(sizeof(std::int64_t) * num_intermediates);
+  auto const num_intermediates = expr_linearizer.intermediate_count();
+  // size_per_thread/block, limit_per_block are for shmem (shared memory)
+  auto const size_per_thread = static_cast<int>(sizeof(std::int64_t) * num_intermediates);
```

Comment:
I think the `shmem` prefix was actually helpful here.

Reply:
ok, i will add it back.